cm2350/ui.py

```diff
@@ -610,9 +621,7 @@ def run(self, maxstep=None, follow=True, showafter=True, runTil=None, pause=True
 
             except KeyboardInterrupt:
                 ### pause the emulation
-                emu.halt_time()
-
-                self.printStats(i)
+                print()
```

Comment:
This is... different?

Reply:
ahh, so you moved `halt_time()` lower as the default way to exit, and `printStats(i)` was already there.  ok.